repos/spack_repo/builtin/packages/openmpi/package.py

```diff
@@ -79,10 +79,13 @@ class Openmpi(AutotoolsPackage, CudaPackage, ROCmPackage):
 
     # Current
     version(
-        "5.0.8", sha256="53131e1a57e7270f645707f8b0b65ba56048f5b5ac3f68faabed3eb0d710e449"
-    )  # libmpi.so.40.40.8
+        "5.0.9", sha256="dfb72762531170847af3e4a0f21d77d7b23cf36f67ce7ce9033659273677d80b"
+    )  # libmpi.so.40.40.7
 
     # Still supported
+    version(
+        "5.0.8", sha256="53131e1a57e7270f645707f8b0b65ba56048f5b5ac3f68faabed3eb0d710e449"
+    )  # libmpi.so.40.40.7
```

Comment:
please change from 7 to 8

Reply:
I changed it to 7 because that is what the file name ends up as. Same for 5.0.9. Is this a bug in OpenMPI?

I didn't want to start removing the comments without understanding better, but does the comment with the library name serve a purpose?